packages/api/src/middleware/judgeMiddleware.ts

```diff
@@ -0,0 +1,31 @@
+import { NextFunction, Request, Response } from 'express';
+import { Judge } from '@hangar/database';
+
+/**
+ * Middleware that evaluates the judge's validity
+ *
+ * Paths:
+ *  - Next function invoked: judge was identified and matching object mounted to request
+ *  - 400: Judge not found
+ *  - 500: Error occurred trying to identify
+ */
+
+export const judgeMiddleware = async (req: Request, res: Response, next: NextFunction) => {
+  let judge;
+  try {
+    judge = await req.entityManager.findOne(Judge, { user: req.session?.id });
+  } catch {
+    res.sendStatus(500);
+    return;
+  }
+
+  if (judge) {
+    req.judge = judge;
+  } else {
+    // Judge does not exist in database
+    res.sendStatus(403);
+    return;
+  }
+
+  next();
```

Comment:
Just a slightly cleaner look
```suggestion
  if (!judge) {
    // Judge does not exist in database
    res.sendStatus(403);
    return;
  }
  
  req.judge = judge;
  next();
```

Reply:
got it , cam i commit it from here itself? @SpencerKaiser